build.sbt

```diff
@@ -371,6 +371,19 @@ lazy val `react-native` = project
     }
   )
 
+lazy val gojs = project
+  .enablePlugins(ScalablyTypedConverterPlugin)
+  .configure(baseSettings, browserProject, reactNpmDeps, bundlerSettings)
+  .settings(
+    useYarn := true,
+    webpackDevServerPort := 8010,
```

Comment:
can you pick the next available number? from the readme it looked like it already went to 8018

Reply:
Done, I've set it to 8019.